packages/quantic/force-app/main/default/lwc/quanticFeedbackModalQna/__tests__/quanticFeedbackModalQna.test.js

```diff
@@ -0,0 +1,299 @@
+import QuanticFeedbackModalQna from 'c/quanticFeedbackModalQna';
+import {createElement} from 'lwc';
+
+jest.mock('@salesforce/label/c.quantic_Yes', () => ({default: 'Yes'}), {
+  virtual: true,
+});
+jest.mock('@salesforce/label/c.quantic_No', () => ({default: 'No'}), {
+  virtual: true,
+});
+jest.mock(
+  '@salesforce/label/c.quantic_FeedbackNotSure',
+  () => ({default: 'Not sure'}),
+  {
+    virtual: true,
+  }
+);
+
+const functionsMocks = {
+  handleSubmit: jest.fn(),
+};
+
+const selectors = {
+  answerEvaluationQuestions: 'lightning-radio-group',
+  questionTopicRadio: 'lightning-radio-group[data-name="correctTopic"]',
+  questionHallucinationRadio:
+    'lightning-radio-group[data-name="hallucinationFree"]',
+  questionDocumentedRadio: 'lightning-radio-group[data-name="documented"]',
+  questionReadableRadio: 'lightning-radio-group[data-name="readable"]',
+  documentUrlInput: 'lightning-input[data-name="documentUrl"]',
+  additionalDetailsTextarea: 'lightning-textarea[data-name="details"]',
+  skipButton: 'lightning-button.skip',
+  submitButton: 'lightning-button.submit-feedback',
+  requiredMessages: 'slds-form-element__help',
```

Comment:
this selector doesn't seem used anywhere, is it extra or are you missing something in a test?

Reply:
yea no you're correct, it was before I realized the inputs were just stubs in tests (so even if they are invalid they don't display their required message) I was trying to make a test to make sure they were displayed, I'll remove it.